src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -156,14 +157,14 @@ const styles = StyleSheet.create({
   username: {
     fontFamily: fonts.inter[700],
     fontSize: 12,
-    color: '#828282',
+    color: COLORS.blackgrey,
     lineHeight: 14,
     marginLeft: 16,
   },
   post: {
     fontFamily: fonts.inter[400],
     fontSize: 16,
-    color: '#333333',
+    color: COLORS.mine_shaft,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the previous one, replacing hard-coded color values with dynamic ones from a theme object. This is a good practice.
```diff
-    color: '#333333',
+    color: COLORS.mine_shaft,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->